### PR TITLE
Saving Clearing interval to prefs

### PR DESCRIPTION
**Pull request by BartSoftware**

### CODE REVIEW

File: app/src/main/java/com/invisibleteam/goinvisible/model/ClearingInterval.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import java.util.concurrent.TimeUnit;
+
+public enum ClearingInterval {
+
+    DAY(R.string.interval_value_day, TimeUnit.DAYS.toMillis(1)),
+    WEEK(R.string.interval_value_week, TimeUnit.DAYS.toMillis(7)),
+    MONTH(R.string.interval_value_month, TimeUnit.DAYS.toMillis(28));
+
+    private final int intevalValueResId;
```
Comment (kejmil01) [inline app/src/main/java/com/invisibleteam/goinvisible/model/ClearingInterval.java]: intevalValueResId -> intervalValueResId

File: app/src/main/res/layout/settings_activity.xml [hunk 2]
```diff
@@ -58,7 +60,7 @@
                 <TextView
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:text="@string/enable_periodic_clearing"
+                    android:text="@{(TextUtils.isEmpty(viewModel.intervalName.get())) ? @string/interval_value_day: viewModel.intervalName.get()}"
```
Comment (kejmil01) [inline app/src/main/res/layout/settings_activity.xml]: Put space between value and colon - @string/interval_value_day:

Comment (michal1990) [inline app/src/main/res/layout/settings_activity.xml]: .get() is useless

Comment (BartSoftware) [reply app/src/main/res/layout/settings_activity.xml]: For condition its necessary

File: app/src/main/res/layout/settings_activity.xml [hunk 3]
```diff
@@ -103,7 +105,7 @@
                 android:layout_margin="@dimen/text_view_margin"
                 android:layout_weight="1"
                 android:enabled="@{viewModel.isClearServiceEnabled}"
-                android:text="@string/interval_value_1_day"
+                android:text="@{(TextUtils.isEmpty(viewModel.intervalName.get())) ? @string/interval_value_day : viewModel.intervalName}"
```
Comment (kejmil01) [inline app/src/main/res/layout/settings_activity.xml]: viewModel.intervalName.get() -> viewModel.intervalName

**Review by kejmil01 — APPROVED** [review]

**Pull request closed without merging**